Add-ons/Umbraco-Deploy/Extending/index.md

```diff
@@ -289,6 +289,16 @@ Examples of these can be found in the open-source Umbraco.Deploy.Contrib project
 
 With the artifact and connectors in place, the final step necessary is to register your entity for deployment.
 
+### Custom Entity Types
+
+If you have introduced custom entity types that will be handled by Umbraco Deploy, they need to be registered with Umbraco so that it can parse the UDI references.
+
+This is done via the following code, which can be triggered from an Umbraco component or an `UmbracoApplicationStartingNotification` handler.
+
+```C#
+UdiParser.RegisterUdiType("mypackage-example", UdiType.GuidUdi);
+```
```

Comment:
```suggestion
### Custom Entity Types

If you have introduced custom entity types that will be handled by Umbraco Deploy, they need to be registered with Umbraco so that it can parse the UDI references.

This is done via the following code, which can be triggered from a Umbraco component or an `UmbracoApplicationStartingNotification` handler.

```C#
UdiParser.RegisterUdiType("mypackage-example", UdiType.GuidUdi);
```
```